config_parser_test.cc

```diff
@@ -76,3 +69,12 @@ TEST_F(NginxStringConfigTest, BlankConfig) {
 TEST_F(NginxStringConfigTest, CommentsOnlyConfig) {
   EXPECT_FALSE(ParseString("# This comment does nothing"));
```

Comment:
That's a good idea! I've added failure messages with https://github.com/KyCodeHuynh/nginx-configparser/pull/1/commits/b384e6665396a45de11232f84b229aa3a625e50f.